package.json

```diff
@@ -106,6 +106,7 @@
     ]
   },
   "dependencies": {
+    "@solana/spl-token-registry": "^0.2.164",
```

Comment:
Why?

Reply:
Oh, I'm sorry. I didn't know, I think it occurred from 'npm i' and Doesn't include on gitignore.